src/components/SingleChoiceListItem.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { capitalizeFirstLetter, getArticleColor } from '../utils/helpers'
+import { ARTICLES } from '../constants/data'
+import { COLORS } from '../constants/colors'
+import styled from 'styled-components/native'
+
+const StyledContainer = styled.View`
+  height: 55px;
+  margin-bottom: 1.5%;
+  border-radius: 2px;
+  border-width: 1px;
+  border-style: solid;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: baseline;
+  border-color: ${(props: any) => {
+    if (props.pressed || props.selected) {
+      return 'transparent'
+    } else {
+      return COLORS.lunesBlackUltralight
+    }
+  }};
+  background-color: ${(props: any) => {
+    if (props.pressed) {
+      return COLORS.lunesBlack
+    } else if (props.selected && props.correct) {
+      return COLORS.lunesFunctionalCorrectDark
```

Comment:
Naming the colors like this is bad practice. If we have design changes, e.g. the now lunesFunctionalCorrectDark becomes light, we have to rename this. When we have named it just after its funtion, e.g. lunesFunctionalCorrectSelected, we only have to change the color value. 
Same for other color occurences.

Reply:
added a new issue for this as the colors weren't defined in this issue.